examples/local/101_initial_cluster.sh

```diff
@@ -37,6 +37,8 @@ for i in 100 101 102; do
 	CELL=zone1 KEYSPACE=commerce TABLET_UID=$i ./scripts/vttablet-up.sh
 done
 
+./scripts/vtorc-up.sh
+
 # set one of the replicas to master
 vtctlclient InitShardMaster -force commerce/0 zone1-100
```

Comment:
This step can be removed, because `vtorc` should do this. But we may have to wait for the master to be elected before the next steps.